mk_verificator/utils/__init__.py

```diff
@@ -23,17 +23,17 @@ def get_active_nodes(config):
 
 
 def get_groups(config):
+    # assume that node name is like <name>.domain
+    # last 1-3 digits of name are index, e.g. 001 in cpu001
+    # name doesn't contain dots
     active_nodes = get_active_nodes(config)
     skipped_group = config.get('skipped_group') or []
     groups = []
-
     for node in active_nodes:
-        group_name = "{group_name}*".format(
-            group_name=node.split('-')[0]
-        )
+        index = re.search('[0-9]{1,3}$', node.split('.')[0])
+        group_name = node.split('.')[0][:-len(index.group(0))]
         if group_name not in skipped_group and group_name not in groups:
```

Comment:
Pls check a section for ceph - line 59.
These is the splitting ceph and ceph-mon in our cloud.
Probably we don't need it now.